game-of-life-native/Game of Life/GameOfLifeViewController.swift

```diff
@@ -11,59 +11,104 @@ import Foundation
 import KotlinGameOfLife
 
 class GameOfLifeViewController: UIViewController, KGOLAppView {
-    
-    private var appPresenter : KGOLAppPresenter?
-    private let controllButton = UIButton(frame: CGRect(x: 100, y: 100, width: 200, height: 50))
-    private let boardView = UIBoard(frame: CGRect(x: 0, y: 150, width: 300, height: 300))
-    
+
+    private let controlButton = UIButton(type: UIButtonType.system)
+    private let patternSelectionButton = UIButton(type: UIButtonType.system)
```

Comment:
you do not really need to init the UIButton with system style, if you do not specify a style that one is the default. Also you can init it like this too `UIButton(type: .system)`

Reply:
If I dont specify it the text becomes invisible, so it doesn't seem to be the default for me 🤔